src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid implements Figure {
```

Comment:
The class should not start with an empty line. Please remove the empty line to follow best practices.